### PR TITLE
Add Stop Time Interpolation

### PR DESCRIPTION
**Pull request by philip-cline**
### Checklist

- [x] Appropriate branch selected _(all PRs must first be merged to `dev` before they can be merged to `master`)_
- [x] Any modified or new methods or classes have helpful JSDoc and code is thoroughly commented
- [x] The description lists all applicable issues this PR seeks to resolve
- [x] The description lists any configuration setting(s) that differ from the default settings
- [x] All tests and CI builds passing

### Description

Front end PR to add support for stop time interpolation in the NormalizeStopTimes modal. 
Backend PR: https://github.com/ibi-group/datatools-server/pull/572
GTFS-lib PR: https://github.com/conveyal/gtfs-lib/pull/399

### CODE REVIEW

**Comment by philip-cline** [comment]
Note that in https://github.com/ibi-group/datatools-ui/pull/998/commits/62216cacba02f4596e040fa10b3334b2de985a52 I did some i18n but there remains a TODO to figure out how to apply `strong` formatting inside of i18n strings

**Review by AdrianaCeric — APPROVED** [review]
Great change!

File: lib/editor/actions/tripPattern.js [hunk 1]
```diff
@@ -51,12 +51,12 @@ export type EditorTripPatternActions = ActionType<typeof normalizeStopTimes> |
...
  * (e.g., a pattern stop is inserted, removed, or its travel times modified).
  */
-export function normalizeStopTimes (patternId: number, beginStopSequence: number) {
+export function normalizeStopTimes (patternId: number, beginStopSequence: number, interpolateStopTimes: boolean) {
   return function (dispatch: dispatchFn, getState: getStateFn) {
     const {data} = getState().editor
     const {feedSourceId} = data.active
     const sessionId = data.lock.sessionId || ''
-    const url = `/api/editor/secure/pattern/${patternId}/stop_times?feedId=${feedSourceId || ''}&sessionId=${sessionId}&stopSequence=${beginStopSequence}`
+    const url = `/api/editor/secure/pattern/${patternId}/stop_times?feedId=${feedSourceId || ''}&sessionId=${sessionId}&stopSequence=${beginStopSequence}&interpolateStopTimes=${interpolateStopTimes.toString()}`
```
Comment (miles-grant-ibigroup) [inline lib/editor/actions/tripPattern.js:59]: This is a crazy hack but glad it works

**Review by miles-grant-ibigroup — APPROVED** [review]
Seems simple enough although there are some strings you haven't localized. Is that intentional?

**Comment by philip-cline** [comment]
Yeah that's intentional, there are `<strong>`, etc. tags in the text and we don't have an easy way of inserting those into i18n strings rn in Datatools

**Pull request merged**